public/electron.js

```diff
@@ -115,7 +115,7 @@ function quitAll() {
 // Boilerplate code - probably just quits the app when all windows are closed
 app.on('window-all-closed', () => {
   if (process.platform !== 'darwin') {
-    quitAll();
+    app.quitAll();
```

Comment:
sure it is not just quitAll? because that is a function we have defined

Reply:
The linter gave me an error without :o Where is it defined?

Reply:
two lines above??

Reply:
The linter does not complain when I remove app. and only keep quitAll()